frontend/test/metabase/scenarios/binning/sql.cy.spec.js

```diff
@@ -13,12 +14,20 @@ const questionDetails = {
 };
 
 describe("scenarios > binning > from a saved sql question", () => {
+  before(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.createNativeQuestion(questionDetails, { loadMetadata: true });
+
+    snapshot("binningSql");
+  });
+
   beforeEach(() => {
     cy.intercept("POST", "/api/dataset").as("dataset");
```

Comment:
It is important to keep all `intercept`s inside a `beforeEeach` hook if we want to have them available in all tests.